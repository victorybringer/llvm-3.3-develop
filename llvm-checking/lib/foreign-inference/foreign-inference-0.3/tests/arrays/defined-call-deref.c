extern int v;
void g(int * arr, int * x) {
  v = arr[*x];
}

int f(int * arr, int x) {
  g(arr, &x);
  return x;
}
