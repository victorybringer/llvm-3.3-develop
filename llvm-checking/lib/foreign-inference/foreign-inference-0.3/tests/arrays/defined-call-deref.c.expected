fromList [ (("@g", "%arr"), 1)
         , (("@f", "%arr"), 1)
         ]
